frontend/src/pages/ArtItemPage/CreateArtItemPage.js

```diff
@@ -1,11 +1,135 @@
-import React from "react";
+import React, { useReducer } from "react";
+import { useAuth } from "../../auth/useAuth";
 
-function CreateArtItemPage() {
+import { TextField, Typography, Stack} from "@mui/material";
+import GenericCardLayout from "../../layouts/GenericCardLayout";
+import ImageUploader from '../../components/ImageUploader';
+import LoadingButton from '../../components/LoadingButton';
+
+function CreateArtItemForm(props) {
+
+    const [formInput, setFormInput] = useReducer(
```

Comment:
Will we do any input validation for these? If then we can use `yup` and `formik` like the implementation in `SignUpPage`

Reply:
I think we don't need to do validation in this form. I only updated the loading button to disable the button unless there is an image.